### ISSUE TITLE
[Exception][WebStorm 2023.2.2] com.intellij.diagnostic.PluginException: ID 'Commit.Button' is already taken by action...

### ISSUE THREAD

**Issue by vjlanguage** [Codium-ai/codiumai-jetbrains-release#117]
# User Description
_User did not enter any detailed information._

# Stack Trace
```
com.intellij.diagnostic.PluginException: ID 'Commit.Button' is already taken by action 'Create commit message (Create commit message)' (class com.intellij.openapi.actionSystem.impl.ChameleonAction)  (Plugin: Git Commit Template). Action 'Generate Commit Message (null)' (class com.intellij.openapi.actionSystem.ActionStub) cannot use the same ID codiumai.codiumai [Plugin: codiumai.codiumai]
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.reportActionIdCollision(ActionManagerImpl.kt:979)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerAction(ActionManagerImpl.kt:918)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerOrReplaceActionInner(ActionManagerImpl.kt:470)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.processActionElement(ActionManagerImpl.kt:443)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerPluginActions(ActionManagerImpl.kt:272)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.registerActions$intellij_platform_ide_impl(ActionManagerImpl.kt:145)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.<init>(ActionManagerImpl.kt:118)
	at com.jetbrains.rdserver.ui.actionPopupMenu.BackendActionManager.<init>(BackendActionManager.kt:15)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
	at com.intellij.serviceContainer.ConstructorInjectionKt.instantiateUsingPicoContainer(constructorInjection.kt:63)
	at com.intellij.serviceContainer.ComponentManagerImpl.instantiateClassWithConstructorInjection(ComponentManagerImpl.kt:1011)
	at com.intellij.serviceContainer.ServiceComponentAdapter.createAndInitialize(ServiceComponentAdapter.kt:47)
	at com.intellij.serviceContainer.ServiceComponentAdapter.doCreateInstance(ServiceComponentAdapter.kt:43)
	at com.intellij.serviceContainer.BaseComponentAdapter.doCreateInstance(BaseComponentAdapter.kt:158)
	at com.intellij.serviceContainer.BaseComponentAdapter.createInstance(BaseComponentAdapter.kt:137)
	at com.intellij.serviceContainer.BaseComponentAdapter.access$createInstance(BaseComponentAdapter.kt:21)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invokeSuspend(BaseComponentAdapter.kt:185)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at com.intellij.serviceContainer.BaseComponentAdapter$getInstanceAsync$2.invoke(BaseComponentAdapter.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.serviceContainer.BaseComponentAdapter.getInstanceAsync(BaseComponentAdapter.kt:180)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsyncIfDefined(ComponentManagerImpl.kt:646)
	at com.intellij.serviceContainer.ComponentManagerImpl.getServiceAsync(ComponentManagerImpl.kt:637)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invokeSuspend(appServicePreloading.kt:179)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2$7.invoke(appServicePreloading.kt)
	at kotlinx.coroutines.intrinsics.UndispatchedKt.startUndispatchedOrReturn(Undispatched.kt:78)
	at kotlinx.coroutines.BuildersKt__Builders_commonKt.withContext(Builders.common.kt:167)
	at kotlinx.coroutines.BuildersKt.withContext(Unknown Source)
	at com.intellij.diagnostic.TracerKt.subtask(tracer.kt:45)
	at com.intellij.diagnostic.TracerKt.subtask$default(tracer.kt:34)
	at com.intellij.ide.bootstrap.AppServicePreloadingKt$preloadCriticalServices$2.invokeSuspend(appServicePreloading.kt:105)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)

```

# Other Information
```
=== About ===
Build version: WebStorm 2023.2.2 Build: #WS-232.9921.42 September 12, 2023
Theme: Gerry Dark
JRE: 17.0.8+7-b1000.22, JetBrains s.r.o.
JVM: 17.0.8+7-b1000.22, OpenJDK 64-Bit Server VM, JetBrains s.r.o.
Operating System: Windows 11.0 (amd64)

=== Plugins ===
Custom plugins: [Regex Rename Files (1.2.5), Gitmoji Plus: Commit Button (1.13.0), Fast Mouse Scrolling (1.6.1), Gerry Themes (2023.1.70), React Component Creator (1.0.1), Batch Scripts Support (1.0.13), Prisma ORM (232.9559.10), String Manipulation (9.11.3), Extra ToolWindow Colorful Icons (1.27.0), Codeium: AI Autocomplete and Chat for Python, JS, TS, Java, Go… (1.2.87), GitHub Markdown Emojis (1.3.2), Database Tools and SQL for WebStorm (232.9921.42), Nyan Progress Bar (1.14), Uniapp-Macro-Support (0.0.6), Key Promoter X (2023.2.0), CodiumAI - Integrity Agent powered by GPT-3.5&4 (0.6.06), Inspection Lens (1.3.0), Vlang (0.0.1-beta.4), Context Free Grammar (0.3.1), CMD Support (1.0.5), Browserslist (1.0.2), Rainbow Brackets (2023.3.4), Ini (232.9921.36), Nginx Configuration (2022.1.1), Gruvbox - Theme (0.6.1), LogIt (v2021.1.2), File Expander (2.2), Grep Console (12.21.0-IJ2021.1), .ignore (4.5.1), Git Commit Template (1.2.0), React Native Snippets (0.1), HighlightBracketPair (1.4.1), TSReact (1.4.0), GitToolBox (500.0.18+222), Vite Integrated (0.2.2), Perl (2023.2), React snippets (1.1.1), .env files support (2023.2), Translation (3.5.4)]
Disabled plugins:[Kubernetes (232.9921.28), Import from Postman Collections (1.0.0-rc.3)]
```

┆Issue is synchronized with this [Trello card](https://trello.com/c/Zl5pxef0) by [Unito](https://www.unito.io)

**Comment by aldanchenko** [comment]
Hi @vjlanguage 

Thanks for reporting!

**Comment by aldanchenko** [comment]
cc @ArtsiomCh

**Comment by ArtsiomCh** [comment]
Fixed in 0.6.07
